docs/README.TXT

```diff
@@ -58,7 +58,7 @@ Essential CC related information and free (with source) tools.
 
 Encoding
 --------
-This version, in both its Linux and Windows builds generates by
+This version, in both its Linux and Windows build generates by
```

Comment:
That's not a typo - build here is a noun, not a verb and there's two builds - the Linux build and the Windows build. We could use binaries instead, of versions. But replacing builds with build is not correct.